10/architectures

```diff
@@ -1,8 +1,8 @@
-bashbrew-arch   variants
-amd64    stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10,alpine3.11
-arm32v6  alpine3.9,alpine3.10,alpine3.11
-arm32v7  stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10,alpine3.11
-arm64v8  stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10,alpine3.11
-i386     alpine3.9,alpine3.10,alpine3.11
-ppc64le  buster,buster-slim,alpine3.9,alpine3.10,alpine3.11
-s390x    buster,buster-slim,alpine3.9,alpine3.10,alpine3.11
+bashbrew-arch  variants
+amd64          alpine3.10,alpine3.11,alpine3.9,buster,buster-slim,stretch,stretch-slim
+arm32v6        alpine3.10,alpine3.11,alpine3.9
```

Comment:
Should 3.9 be in front, or does the alphasort keep the 9 after the 1s?

Reply:
The basic alphasort orders them like this since 1 < 9 at that char.